presentation/README.md

```diff
@@ -0,0 +1,433 @@
+  public void loadFromPMod() {
+    LOGGER.info("Load data from PresentationModel");
+    txtArtist.setText(model.getArtist());
+    txtTitle.setText(model.getTitle());
+    chkClassical.setSelected(model.getIsClassical());
+    txtComposer.setEditable(model.getIsClassical());
+    txtComposer.setText(model.getComposer());
+  }
+
+  /**
+   * initialize the GUI.
+   */
+  public void createView() {
+    var frame = new JFrame("Album");
+    var b1 = Box.createHorizontalBox();
+
+    frame.add(b1);
+    albumList = new JList<>(model.getAlbumList());
+    albumList.addMouseListener(new MouseAdapter() {
+      @Override
+      public void mouseClicked(final MouseEvent e) {
+        model.setSelectedAlbumNumber(albumList.getSelectedIndex() + 1);
+        loadFromPMod();
+      }
+    });
+    b1.add(albumList);
+
+    var b2 = Box.createVerticalBox();
+    b1.add(b2);
+
+    txtArtist = new TextField();
+    txtTitle = new TextField();
+
+    final var widthTxt = 200;
+    final var heightTxt = 50;
+    txtArtist.setSize(widthTxt, heightTxt);
+    txtTitle.setSize(widthTxt, heightTxt);
+
+    chkClassical = new JCheckBox();
+    txtComposer = new TextField();
+    chkClassical.addActionListener(itemEvent -> {
+      txtComposer.setEditable(chkClassical.isSelected());
+      if (!chkClassical.isSelected()) {
+        txtComposer.setText("");
+      }
+    });
+    txtComposer.setSize(widthTxt, heightTxt);
+    txtComposer.setEditable(model.getIsClassical());
+
+    apply = new JButton("Apply");
+    apply.addMouseListener(new MouseAdapter() {
+      @Override
+      public void mouseClicked(final MouseEvent e) {
+        saveToPMod();
+        loadFromPMod();
+      }
+    });
+    cancel = new JButton("Cancel");
+    cancel.addMouseListener(new MouseAdapter() {
+      @Override
+      public void mouseClicked(final MouseEvent e) {
+        loadFromPMod();
+      }
+    });
+
+    b2.add(txtArtist);
+    b2.add(txtTitle);
+
+    b2.add(chkClassical);
+    b2.add(txtComposer);
+
+    b2.add(apply);
+    b2.add(cancel);
+
+    final var x = 200;
+    final var y = 200;
+    final var width = 500;
+    final var height = 300;
+    frame.setDefaultCloseOperation(javax.swing.WindowConstants.EXIT_ON_CLOSE);
+    frame.setBounds(x, y, width, height);
+    frame.setVisible(true);
+  }
+
+}
+
+```
+
+Class `Album` is to store information of a album.
+
+```java
+@Setter
+@Getter
+@AllArgsConstructor
+public class Album {
+  /**
+   * the title of the album.
+   */
+  private String title;
+  /**
+   * the artist name of the album.
+   */
+  private String artist;
+  /**
+   * is the album classical, true or false.
+   */
+  private boolean isClassical;
+  /**
+   * only when the album is classical,
+   * composer can have content.
+   */
+  private String composer;
+}
+
+```
+
+Class `DsAlbum` is store the information of all the albums that will be shown on GUI.
+
+```java
+/**
+ * a class used to deal with albums.
+ */
+@Slf4j
+@Getter
+public class DsAlbum {
```

Comment:
Could we reconsider the naming of this class? It's not very clear what this Ds in front means...